app/post.go

```diff
@@ -798,20 +799,32 @@ func (a *App) parseAndFetchChannelIdByNameFromInFilter(channelName, userId, team
 }
 
 func (a *App) searchPostsInTeam(teamId string, userId string, paramsList []*model.SearchParams, modifierFun func(*model.SearchParams)) (*model.PostList, *model.AppError) {
-	channels := []store.StoreChannel{}
+	var wg sync.WaitGroup
+
+	//create a channel with buffer size of len(paramsList)
```

Comment:
nit: this comment is redundant -- can we remove it? Comments are best when the add context that a plain reading of the code doesn't offer (e.g. architecture or edge case considerations).

Reply:
```suggestion
```